_alerts/2017-10-09-segwit2x-safety.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Beware of Bitcoin's possible incompatibility with some major services"
+shorturl: "segwit2x-safety"
+active: true
+banner: "Beware of Bitcoin's possible incompatibility with some major services"
+bannerclass: "alert"
+date: 2017-10-11
+---
+
+## Summary
+
+A subsection of the Bitcoin economy including prominent businesses such as Coinbase, Xapo, and BitPay have signed an agreement to adopt
+and implement a contentious hard fork of Bitcoin [sometime](http://bashco.github.io/2x_Countdown/) in November. A hard fork is a
+backwards incompatible change to the Bitcoin network. This hard fork is not supported by
+the majority of the Bitcoin users and developers and is therefore a contentious hard fork.
+By adopting this hard fork, we believe the supporters of this agreement are shifting their users to an alternative currency (an altcoin) which is incompatible with Bitcoin.
+
+The signatories of this agreement wrongly believe that the currency created by adopting
+this contentious hard fork will eventually become Bitcoin. Therefore storing any BTC on services such
+as Coinbase, Bitpay and Xapo is strongly not recommended. By storing BTC on these services, you could
+find that after the hard fork, your BTC has been renamed to something else or replaced entirely with the new altcoin.
+The best way to ensure that your BTC is protected is to [download the latest version of Bitcoin Core](https://bitcoin.org/en/download)
+and transfer out any BTC stored on services that are a signatory to this agreement. We have
+listed the businesses supporting this contentious hard fork at the bottom of this document.
+
+For users who aren't prepared to install Bitcoin Core and go through the lengthy set up process, we recommend
+a wallet such as [GreenAddress](https://greenaddress.it/) or [Electrum](https://electrum.org/). Avoid using web wallets like blockchain.info.
+However users should only use these wallets to store their coins and never perform transactions until well after the hard fork. Any transactions you
+make on the Bitcoin blockchain can also be replicated and "replayed" on the altcoin chain. If the coins on the contentious
+hard fork have any value, there will be methods you can use to "split" your coins and have access to them. Pay extra attention
+to major Bitcoin communication channels and media shortly after the fork so you stay informed.
+
+## Mobile wallets
+
+Miners in Bitcoin are responsible for transaction ordering through putting transactions into blocks, and collectively
+creating a chain of these blocks. Most mobile wallets depend on blindly trusting the longest chain produced by the miners in order to safely send and receive transactions.
+There is considerable mining power supporting the contentious hard fork. A supermajority of miners have pledged to support the contentious
```

Comment:
I don't think it can be considered a supermajority of miners anymore. It's just a majority now.